src/intro/useCases/utils/__tets__/utils.test.ts

```diff
@@ -0,0 +1,16 @@
+import { describe, expect, it } from 'vitest';
+import { getGreeting } from '../getGreeting';
+
+describe('utils', () => {
+    describe('getGreeting', () => {
+        it("should return 'Good morning' when hour is less than 12", () => {
+            const result = getGreeting(11);
+            expect(result).toBe('Good morning');
+        });
+
+        it("should return 'Hello' when hour is greater than or equal to 12", () => {
+            const result = getGreeting(12);
+            expect(result).toBe('Hello');
```

Comment:
When the test case employs AAA formation, it simplifies cognitive load when looking at it. Especially, for test cases that will be more complex than this trivial one.

Each A is separated from the next with a blank line:

```suggestion
            // (A)rrangement
            const hour = 12; 

            // (A)ction
            const result = getGreeting(12);
            
            // (A)ssertion
            expect(result).toBe('Hello');
```
_(obviously these comments are unnecessary, I used them to demonstrate the AAA formation for you)_